### PR TITLE
Tenancy support in AKO

### PR DESCRIPTION
**Pull request by parimanur**

### CODE REVIEW

File: docs/AKO/README.md [hunk 1]
```diff
@@ -181,6 +183,28 @@ A new parameter serviceType has been introduced as config option in AKO's values
 
 Kubernetes populates NodePort by default for service of type LoadBalancer. If config.serviceType is set to NodePort, AKO would use NodePort as backend for service of type Loadbalancer instead of using Endpoints, which is the default behaviour with config.serviceType set as ClusterIP.
 
+### Tenancy support in AKO
+
+This feature allows AKO to map each kubernetes / OpenShift cluster uniquely to a tenant in Avi. `ControllerSettings.tenantsPerCluster` needs to be set to `true` to  enable this feature.
+
+#### Steps enable Tenancy in AKO
```
Comment (sudswasavi) [inline docs/AKO/README.md:190]: nit: Steps to

**Review by sudswasavi — COMMENTED** [review]
Great work @parimanur . One question though - when AKO is used for NodePort - do you still require the VRF permission? If not, could you also put that in the doc?

**Review by monotosh-avi — APPROVED** [review]

File: docs/AKO/README.md (hunk 1 above)
Comment (parimanur) [reply docs/AKO/README.md]: Addressed Thanks!

**Comment by parimanur** [comment]
> Great work @parimanur . One question though - when AKO is used for NodePort - do you still require the VRF permission? If not, could you also put that in the doc?

Yes, I have added a Note for the VRFContext permission in NodePort.

**Pull request merged**